src/main/java/net/masterthought/cucumber/ReportInformation.java

```diff
@@ -1,11 +1,6 @@
 package net.masterthought.cucumber;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-
+import com.googlecode.totallylazy.Sequence;
```

Comment:
try to avoid changing order of the imports - otherwise next commit will change this order back